source/leuven/services/openondemand.rst

```diff
@@ -38,7 +36,7 @@ This menu provides a file explorer that allows you to navigate files by clicking
 available here. General file explorer options like moving, deleting, modifying and creating files or directories are available as well. You can also use this interface
 to download and upload files to and from your local machine. Be aware that this is not recommended for very large files.
 
-**Good to know:** the standard 'ctrl+s' sadly does not store your files on Open OnDemand, but will trigger a save on your local machine. Luckily, there is a quite 
+**Good to know:** the standard 'ctrl+s' does not save your edited files on Open OnDemand, but will trigger a save on your local machine. Luckily, there is a quite 
 obvious save button on the editor page.
```

Comment:
The save button is definitely there but it may not be all that obvious to the users. At first I was also a bit confused with this because it appears to be greyed-out to eventhough it is functioning just fine. It does not really help much that the button does not give any kind of feedback when being clicked,.